### ISSUE TITLE
Hex complains about Elixir version needlessly

### ISSUE THREAD

**Issue by opie4624** [hexpm/hex#206]
```
> mix hex.info
Hex:    0.11.0
Elixir: 1.2.3
OTP:    18.2.1

Built with: Elixir 1.2.3 and OTP 18.2.4

Hex was built against against Elixir 1.2.3 and you are running 1.2.3, please run `mix local.hex` to update to a matching version
Registry file available (last updated: 2016-03-03 01:01:26)
Size: 1572kB (compressed 511kb)
Packages #: 1642
Versions #: 7323
```

**Comment by jbavari** [comment]
I'm running into the same error. Before the hex update, everything was working fine. Now I'm getting local crashes, as well as our CI running into the same errors:

```
mix do deps.get, deps.compile
* Updating ueberauth (https://github.com/ueberauth/ueberauth.git)
* Updating exzmq (https://github.com/jbavari/exzmq)
* Updating gen_listener_tcp (https://github.com/kaos/gen_listener_tcp.git)
Hex was built against against Elixir 1.1.1 and you are running 1.1.1, please run `mix local.hex` to update to a matching version
Running dependency resolution
** (Mix) Invalid requirement ">=1.1.1" defined for package uuid
```

```
mix hex.info
Hex:    0.11.0
Elixir: 1.1.1
OTP:    18.2.1

Built with: Elixir 1.1.1 and OTP 17.5.6.7

Hex was built against against Elixir 1.1.1 and you are running 1.1.1, please run `mix local.hex` to update to a matching version
Registry file available (last updated: 2016-03-03 01:19:59)
Size: 1572kB (compressed 511kb)
Packages #: 1642
Versions #: 7323
```

**Comment by ericmj** [comment]
Both issues have been fixed in 0.11.1
